### PR TITLE
chore: update tw intellisense

### PR DESCRIPTION
**Pull request by alantoa**
# Why

- [prettier-plugin-tailwindcss](https://github.com/tailwindlabs/prettier-plugin-tailwindcss) support[ custom attributes PR](https://github.com/tailwindlabs/prettier-plugin-tailwindcss/pull/155) merged, so we don't need to use [this patch](https://github.com/showtime-xyz/showtime-frontend/blob/staging/patches/prettier-plugin-tailwindcss%2B0.2.5.patch) anymore!
- update TW intellisense. It now supports all `tw` regex intellisense.

image

image

image

# How

# Test Plan

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -8,13 +8,7 @@
     // already have a Prettier plugin, so don't need this.
```
Comment (hirbod) [inline .vscode/settings.json:8]: Comments are not allowed in .json. This could / should actually fail

File: package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "nativewind": "^2.0.11",
     "patch-package": "^7.0.0",
     "prettier": "^2.8.7",
-    "prettier-plugin-tailwindcss": "0.2.5",
+    "prettier-plugin-tailwindcss": "insiders",
```
Comment (hirbod) [inline package.json:24]: What is this? Never seen before

**Review by hirbod — CHANGES_REQUESTED** [review]
LGTM, but comments should be removed

File: .vscode/settings.json (hunk 1 above)
Comment (alantoa) [reply .vscode/settings.json:8]: No, now it's the age of JSON5 and we are using it. This is not a problem.

Comment (hirbod) [reply .vscode/settings.json:8]: This does not work in every env though. But yeah, works in VSCode.

**Review by hirbod — APPROVED** [review]
LGTM

**Review by hirbod — APPROVED** [review]
LGTM

**Pull request merged**